Test failure: node_integration_tests::test_exr

```____________________________ GolemNodeTest.test_exr ____________________________
self = <scripts.node_integration_tests.tests.test_golem.GolemNodeTest testMethod=test_exr>
    @unittest.skipIf(PROTOCOL_CONST.ID <= '29', "Known issue in 0.18.x")
    def test_exr(self):
        exit_code = self._run_test('golem.exr')
>       self.assertEqual(exit_code, 0)
E       AssertionError: 1 != 0
```

Seems to be caused by:

```
Task dict: {'type': 'Blender', 'name': 'test task', 'timeout': '0:10:00', 'subtask_timeout': '0:09:50', 'subtasks_count': 1, 'bid': 1.0, 'resources': ['/home/buildbot-worker/worker/test_node_integration/build/scripts/node_integration_tests/tasks/test_task_1/puchacz_canopy.jpg', '/home/buildbot-worker/worker/test_node_integration/build/scripts/node_integration_tests/tasks/test_task_1/wlochaty3.blend'], 
'options': {'output_path': '/tmp/golem-integration-test-output-v9ylx8cj', 'format': 'PNG', 'resolution': [320, 240]}}
```

Looks like `options.format = 'PNG'`

[source build log](https://buildbot.golem.network/buildbot/#/builders/15/builds/635/steps/4/logs/stdio)

--- a/scripts/node_integration_tests/playbooks/golem/exr.py
+++ b/scripts/node_integration_tests/playbooks/golem/exr.py
@@ -3,5 +3,4 @@ from ..test_config_base import TestConfigBase
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__()
-        self.task_settings = 'exr'
+        super().__init__('exr')

--- a/scripts/node_integration_tests/playbooks/golem/exr.py
+++ b/scripts/node_integration_tests/playbooks/golem/exr.py
@@ -3,4 +3,4 @@ from ..test_config_base import TestConfigBase
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__('exr')
+        super().__init__(task_settings='exr')

--- a/scripts/node_integration_tests/playbooks/golem/jpeg.py
+++ b/scripts/node_integration_tests/playbooks/golem/jpeg.py
@@ -3,4 +3,4 @@ from ..test_config_base import TestConfigBase
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__('jpeg')
+        super().__init__(task_settings='jpeg')

--- a/scripts/node_integration_tests/playbooks/golem/jpeg.py
+++ b/scripts/node_integration_tests/playbooks/golem/jpeg.py
@@ -3,5 +3,4 @@ from ..test_config_base import TestConfigBase
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__()
-        self.task_settings = 'jpeg'
+        super().__init__('jpeg')

--- a/scripts/node_integration_tests/playbooks/golem/jpg/test_config.py
+++ b/scripts/node_integration_tests/playbooks/golem/jpg/test_config.py
@@ -3,5 +3,4 @@ from ...test_config_base import TestConfigBase
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__()
-        self.task_settings = 'jpg'
+        super().__init__('jpg')

--- a/scripts/node_integration_tests/playbooks/golem/jpg/test_config.py
+++ b/scripts/node_integration_tests/playbooks/golem/jpg/test_config.py
@@ -3,4 +3,4 @@ from ...test_config_base import TestConfigBase
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__('jpg')
+        super().__init__(task_settings='jpg')

--- a/scripts/node_integration_tests/playbooks/golem/restart_frame/test_config.py
+++ b/scripts/node_integration_tests/playbooks/golem/restart_frame/test_config.py
@@ -11,4 +11,3 @@ class TestConfig(TestConfigBase):
             requestor_config,
             requestor_config_2,
         ]
-        self.task_settings = 'default'

--- a/scripts/node_integration_tests/playbooks/golem/task_timeout/test_config.py
+++ b/scripts/node_integration_tests/playbooks/golem/task_timeout/test_config.py
@@ -3,7 +3,7 @@ from ...test_config_base import TestConfigBase, make_node_config_from_env
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__('2_short')
+        super().__init__(task_settings='2_short')
         provider_config = make_node_config_from_env('PROVIDER', 1)
         provider_config.script = 'provider/no_wtct_after_ttc'
         provider_config_2 = make_node_config_from_env('PROVIDER', 1)

--- a/scripts/node_integration_tests/playbooks/golem/task_timeout/test_config.py
+++ b/scripts/node_integration_tests/playbooks/golem/task_timeout/test_config.py
@@ -3,7 +3,7 @@ from ...test_config_base import TestConfigBase, make_node_config_from_env
 
 class TestConfig(TestConfigBase):
     def __init__(self):
-        super().__init__()
+        super().__init__('2_short')
         provider_config = make_node_config_from_env('PROVIDER', 1)
         provider_config.script = 'provider/no_wtct_after_ttc'
         provider_config_2 = make_node_config_from_env('PROVIDER', 1)
@@ -11,4 +11,3 @@ class TestConfig(TestConfigBase):
             provider_config,
             provider_config_2,
         ]
-        self.task_settings = '2_short'

--- a/scripts/node_integration_tests/playbooks/test_config_base.py
+++ b/scripts/node_integration_tests/playbooks/test_config_base.py
@@ -55,7 +55,7 @@ def make_node_config_from_env(role: str, counter: int) -> NodeConfig:
 
 
 class TestConfigBase:
-    def __init__(self) -> None:
+    def __init__(self, task_settings: str = 'default') -> None:
         self.dump_output_on_crash = False
         self.dump_output_on_fail = False
 
@@ -65,7 +65,7 @@ class TestConfigBase:
             make_node_config_from_env('PROVIDER', 1)
         self._nodes_index = 0
         self.task_package = 'test_task_1'
-        self.task_settings = 'default'
+        self.task_settings = task_settings
         self.task_dict = helpers.construct_test_task(
             task_package_name=self.task_package,
             task_settings=self.task_settings,

--- a/scripts/node_integration_tests/playbooks/test_config_base.py
+++ b/scripts/node_integration_tests/playbooks/test_config_base.py
@@ -55,7 +55,7 @@ def make_node_config_from_env(role: str, counter: int) -> NodeConfig:
 
 
 class TestConfigBase:
-    def __init__(self, task_settings: str = 'default') -> None:
+    def __init__(self, *, task_settings: str = 'default') -> None:
         self.dump_output_on_crash = False
         self.dump_output_on_fail = False
 

--- a/scripts/node_integration_tests/tests/test_golem.py
+++ b/scripts/node_integration_tests/tests/test_golem.py
@@ -37,6 +37,7 @@ class GolemNodeTest(NodeTestBase, unittest.TestCase):
         exit_code = self._run_test('golem.exr')
         self.assertEqual(exit_code, 0)
 
+    @unittest.skipIf(True, "Disabled until verification is fixed #4143")
     def test_jpeg(self):
         exit_code = self._run_test('golem.jpeg')
         self.assertEqual(exit_code, 0)

